.NET/Microsoft.Recognizers.Text.DateTime/Italian/Parsers/ItalianSetParserConfiguration.cs

```diff
@@ -141,5 +144,43 @@ public bool GetMatchedUnitTimex(string text, out string timex)
 
             return true;
         }
+
+         // for SetWeekDay when the plural is not formed by adding 's'
+        public void SetWeekDayParser(IDateTimeExtractor extractor, string text, Match match, DateObject reference, ref List<ExtractResult> ers, ref bool success)
```

Comment:
Same here. If nothing is parsed, it can return an empty list. The ```success``` condition can be in the calling code.